6_visualize.yml

```diff
@@ -35,6 +34,7 @@ targets:
       - 6_visualize/log/6_storm_gif_tasks.ind
       - 6_intro_gif_tasks.yml
       - 6_visualize/log/6_intro_gif_tasks.ind
+      - 6_visualize/log/6_outro_gif_tasks.ind
       - 6_visualize/out/animation_a.gif
```

Comment:
is there a reason the `6_visualize/out/animation_a.gif` isn't the only depends here? Since the others go into the animation target as inputs

Reply:
No reason that I know of. That can certainly be changed. This is likely just a leftover practice from the original implementation of this pipeline back when I didn't understand them completely :) I'd love to do an overhaul after I finish up Pipelines III ... so many improvements can be made!